packages/esm-appointments-app/src/empty-state/empty-state.component.tsx

```diff
@@ -27,17 +26,6 @@ export const EmptyState: React.FC<EmptyStateProps> = ({ headerTitle, displayText
         <p className={styles.content}>
           {t('emptyStateText', 'There are no {{displayText}} to display', { displayText })}
         </p>
-        {scheduleType === 'Scheduled' ? (
-          <p className={styles.action}>
-            {launchForm && (
-              <span>
-                <Button renderIcon={Add} kind="ghost" onClick={() => launchForm()}>
-                  {t('createAppointment', 'Create appointment')}
-                </Button>
-              </span>
-            )}
-          </p>
```

Comment:
This is the aforemented logic that rendered a "Create appointment" button, but only if there were no scheduled appointments on the current day.  I could see why this is "nice" functionality to have, but we already have a create appointment button on this page.